serde-oracle-jdbc-json/src/test/java/io/micronaut/serde/bson/SampleData.java

```diff
@@ -9,28 +9,20 @@
 import java.util.UUID;
 
 import io.micronaut.core.annotation.Introspected;
-import io.micronaut.serde.oracle.jdbc.json.annotation.OracleType;
-
-import static io.micronaut.serde.oracle.jdbc.json.annotation.OracleType.Type.*;
 
 @Introspected
 public class SampleData {
 
-    @OracleType(TEMPORAL)
```

Comment:
So this is not needed anymore. However, these beans/entities won't work in HTTP client deserialization since it would trigger Oracle serde. So, advice would be to use DTO for transfer over HTTP. Think we don't have other way because Oracle serdes will be in the context.